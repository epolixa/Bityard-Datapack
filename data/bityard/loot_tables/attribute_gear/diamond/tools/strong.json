{
  "pools": [
    {
      "rolls": 1,
      "entries": [
        {
          "type": "item",
          "name": "minecraft:diamond_sword",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Strong Diamond Sword\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -2.4,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 7.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:diamond_pickaxe",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Strong Diamond Pickaxe\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -2.8,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 5.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:diamond_axe",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Strong Diamond Axe\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -3.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 9.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:diamond_shovel",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Strong Diamond Shovel\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -3.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 5.5,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:diamond_hoe",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Strong Diamond Hoe\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 1.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        }
      ]
    }
  ]
}