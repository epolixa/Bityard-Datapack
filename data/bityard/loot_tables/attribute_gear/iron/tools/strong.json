{
  "pools": [
    {
      "rolls": 1,
      "entries": [
        {
          "type": "item",
          "name": "minecraft:iron_sword",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Strong Iron Sword\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -2.4,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 6.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:iron_pickaxe",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Strong Iron Pickaxe\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -2.8,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 4.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:iron_axe",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Strong Iron Axe\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -3.1,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 9.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:iron_shovel",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Strong Iron Shovel\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -3.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 4.5,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:iron_hoe",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Strong Iron Hoe\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -1.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 1.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        }
      ]
    }
  ]
}