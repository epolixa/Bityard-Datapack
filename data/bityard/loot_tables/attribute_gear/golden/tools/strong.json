{
  "pools": [
    {
      "rolls": 1,
      "entries": [
        {
          "type": "item",
          "name": "minecraft:golden_sword",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Strong Golden Sword\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -2.4,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 4.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:golden_pickaxe",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Strong Golden Pickaxe\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -2.8,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 2.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:golden_axe",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Strong Golden Axe\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -3.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 7.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:golden_shovel",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Strong Golden Shovel\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -3.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 2.5,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        },
        {
          "type": "item",
          "name": "minecraft:golden_hoe",
          "weight": 1,
          "functions": [
            {
              "function": "set_nbt",
              "tag": "{loot_type:\"attribute_gear\", display: {Name: \"{\\\"italic\\\":\\\"false\\\", \\\"text\\\":\\\"Strong Golden Hoe\\\"}\"}}"
            },
            {
              "function": "set_attributes",
              "modifiers": [
                {
                  "attribute": "generic.attackSpeed",
                  "name": "Attack Speed",
                  "amount": -3.0,
                  "operation": "addition",
                  "slot": "mainhand"
                },
                {
                  "attribute": "generic.attackDamage",
                  "name": "Attack Damage",
                  "amount": 1.0,
                  "operation": "addition",
                  "slot": "mainhand"
                }
              ]
            }
          ]
        }
      ]
    }
  ]
}